### PR TITLE
[finishes #163753223] created login route and authenticated offices

### PR DESCRIPTION
**Pull request by successgilli**
#### What does this PR do?
It creates a route for the user to login
#### Description of Task to be completed?
Have the endpoint working

- POST api/v1/auth/login
#### How should this be manually tested?
after cloning the repo, cd into it, run 'npm i' and type npm run test.
The endpoint can also be tested using Postman or your browser
when the signup form is posted to the above route, it should return a token for the user.
if email of user exist, and password is correct.
#### Any background context you want to provide?
Not yet
#### What are the relevant pivotal tracker stories?
#163753223
#### Screenshots (if appropriate)
image
authorising the POST /api/v1/offices route
#### Questions:
none

### CODE REVIEW

File: src/controllers/userController.js [hunk 1]
```diff
@@ -14,10 +16,11 @@ class UserController {
         password,
         phoneNumber,
         passportUrl,
+        isAdmin,
       } = req.body;
       const hashedPassword = passwordHash.generate(password);
-      const text = 'INSERT INTO users (firstName, lastName, hashedPassword, otherName, email, phoneNumber, passportUrl) values ($1,$2,$3,$4,$5,$6,$7) RETURNING *';
-      const param = [firstName, lastName, hashedPassword, otherName, email, phoneNumber, passportUrl];
+      const text = 'INSERT INTO users (firstName, lastName, hashedPassword, otherName, email, phoneNumber, passportUrl, isAdmin) values ($1,$2,$3,$4,$5,$6,$7,$8) RETURNING *';
```
Comment (nouwatinjacob) [inline src/controllers/userController.js:22]: Why do you name this `text`? I think you can give it a better for it to be readable. What do you think?

Comment (nouwatinjacob) [reply src/controllers/userController.js:22]: You can abstract all your queries so that you can just import them anywhere needed.

Comment (successgilli) [reply src/controllers/userController.js:22]: it is 'text' not test

Comment (successgilli) [reply src/controllers/userController.js:22]: okay sir

**Pull request merged**